Index: app/src/main/java/com/jack/thehungrydeveloper/MainCoursesActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/jack/thehungrydeveloper/MainCoursesActivity.java	(revision 7ef24028b3415733e89e1d366d372c53d8b52232)
+++ app/src/main/java/com/jack/thehungrydeveloper/MainCoursesActivity.java	(date 1626619744414)
@@ -18,7 +18,14 @@
         Dish[] mainCourses = {
                 new Dish("Onion and parmesan ciabatta", "Warm ciabatta filled with chargrilled onion and parmesan", 124),
                 new Dish("Spinach and banana korma", "Creamy korma made with fresh spinach and banana", 542),
-                new Dish("Pepper and mushroom maki", "Toasted seaweed wrapped around sushi rice, filled with green pepper and crimini mushroom", 153)
+                new Dish("Pepper and mushroom maki", "Toasted seaweed wrapped around sushi rice, filled with green pepper and crimini mushroom", 153),
+                new Dish("Chickpea and parsley wontons", "Thin wonton cases stuffed with fresh chickpea and parsley", 235),
+                new Dish("Apple and oatbran buns", "Moist buns made with crisp apple and oatbran", 732),
+                new Dish("Coriander and strawberry cookies", "Crunchy cookies made with fresh coriander and strawberries", 197),
+                new Dish("Aubergine and chilli dumplings", "Thin pastry cases stuffed with chargrilled aubergine and firey chilli", 185),
+                new Dish("Cardamom and pepper dip", "A dip made from green cardamom and bell pepper", 185),
+                new Dish("Rhubarb and lamb pie", "A shortcrust pasty case filled with fresh rhubarb and minced lamb", 147),
+                new Dish("Bocconcini and spring onion salad", "A crunchy salad featuring bocconcini and fresh spring onion", 194)
         };
     }
 }
